src/pages/index.tsx

```diff
@@ -44,6 +77,25 @@ const Index: FC = () => {
     }
   }, [isLoading, response]);
 
+  useEffect(() => {
+    const url = SELF_URL;
+    const fetchData = async () => {
+      try {
+        const r_esponse = fetch({ url });
+        const fetchPromise = await r_esponse.requestPromise;
+        const adminUser :boolean = fetchPromise.data.user.roles.admin;
+        const superUser :boolean = fetchPromise.data.user.roles.super_user;
```

Comment:
Better to use optional chaining here?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Alright I will go through this and see if it fits the code.

Reply:
Yes this is useful. Thank you

Reply:
Not using it?

Reply:
I am going to use it and update the pr